### PR TITLE
fastconfigure.ui: add notice about encryption

### PR DESCRIPTION
**Pull request by mathiascode**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,8 @@ I stumbled across [this quote](http://www.bash.org/?10159):
 
 Nicotine+ does not collect any user data. Different policies may apply to the default Soulseek server, which is not operated by the Nicotine+ Team.
 
+Soulseek is unencrypted, and should not be used for secure communication. Use a VPN if you are concerned about your IP address being public.
```
Comment (slook) [inline README.md:54]: "should not be used for" -> "is not intended for"

It is not warranted to raise any concerns about the IP address, nor is it our remit to suggest using VPN or other such.

**Review by slook — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (hboetes) [reply README.md:54]: Soulseek is an unencrypted protocol,...

Comment (mathiascode) [reply README.md:54]: > "should not be used for" -> "is not intended for"
> 
> It is not warranted to raise any concerns about the IP address, nor is it our remit to suggest using VPN or other such.

Right, I suppose you can already draw that conclusion from the statement about encryption.

**Comment by mathiascode** [comment]
Updated

**Pull request merged**